test/test_train_mp_imagenet.py

```diff
@@ -132,13 +151,30 @@ def train_imagenet():
 
   torch.manual_seed(42)
 
+  devices = (
```

Comment:
This will always return 1 device.
IOW, do not use that API for this.

Reply:
removed this line